qemu/tests/cfg/cpu_info_check.cfg

```diff
@@ -0,0 +1,10 @@
+- cpu_info_check:
+    type = cpu_info_check
+    start_vm = no
+    cpu_model_3_1_0 = Cascadelake-Server Icelake-Client Icelake-Server KnightsMill
+    cpu_model_4_2_0 = Cascadelake-Server-noTSX Icelake-Client-noTSX Icelake-Server-noTSX
+    cpu_model_4_2_0 += Cascadelake-Server-v1 Cascadelake-Server-v2 Cascadelake-Server-v3
+    cpu_model_4_2_0 += Icelake-Server-v1 Icelake-Server-v2 EPYC-v1 EPYC-v2
+    cpu_model_8_1_0 = Cooperlake Cooperlake-v1 Snowridge Snowridge-v1 Snowridge-v2
```

Comment:
Hi, does `cpu_model_8_1_0` mean rhel.8.2.1 ?

Reply:
@nanliu-r Yes a typo, will update it. Thanks.

Reply:
@nanliu-r  Updated, please help re-review, thanks.